test/test_benchmark.py

```diff
@@ -240,6 +267,16 @@ def get_output_files(self):
     return ret
 
 
+class SizeBenchmarker(EmscriptenBenchmarker):
+  record_stats = True
+
+  def __init__(self, name):
+    super().__init__(name, None, [])
+
```

Comment:
```suggestion
  def __init__(self, name):
    # do not set an engine, as we will not run the code
    super().__init__(name, engine=None, extra_args=[])

```
(why must `extra_args` be set?)